app/helpers/camp_applications_helper.rb

```diff
@@ -0,0 +1,8 @@
+module CampApplicationsHelper
+	def show_confirmation(application, status)
+		if application.progress.equal?(100) || (application.progress.equal?(90) && application.step_ten.blank?)
+			arr = ['confirm', 'You cannot edit application after submission. Are you sure to submit?']
```

Comment:
{ 'confrim' => 'You cannot edit application after submission. Are you sure to submit?' }